recipe_batches/recipe_batches.py

```diff
@@ -1,9 +1,28 @@
 #!/usr/bin/python
 
 import math
+"""
+Took me another 2 hours
+"""
 
 def recipe_batches(recipe, ingredients):
-  pass 
+  batches = [] #create an empty list
+  loop = 1e1000 # create a big min number which can be replaced
+  for k,v in recipe.items(): #for key value pair in dict
+#     print(k,v)
+#     total_ingredient_amt = ingredients[k] #[k] is an index which ouputs value for each key
+      if k in ingredients: # like i in a list
+        batch_per_ingredient = ingredients[k]//recipe[k] #create a list of floor division for whole batches
+        batches.append(batch_per_ingredient) #append it to the empty list
+      else:
+        batches.append(0) #if the lists have discrepency then append 0
```

Comment:
Good job with this one. Way to problem solve!

If there is no k in ingredients, do you really need to append 0 to the batches list? Couldn't you just return 0 because we already know we can't complete the recipe with our current set of ingredients?

Reply:
That's a good point!